README.md

```diff
@@ -87,6 +87,8 @@ Do this to start a GOV.UK web app:
 govuk-docker up collections-publisher-app
 ```
 
+👉 [Replicate data locally](docs/how-tos.md#how-to-replicate-data-locally) (or use the `app-live` stack).
```

Comment:
I feel like it'd be helpful if this linked to #the-app--stacks - I know it's only a couple of lines below but it doesn't seem super clear that contextually that this is what that refers to.

Reply:
Sure, that's fair. I've added that in 👍.